src/scry/implementations.cr

```diff
@@ -1,9 +1,26 @@
 require "./log"
 require "./protocol/location"
+require "./build_failure"
 
 module Scry
   # Using Crystal implementation to emulate GoTo Definition.
   struct Implementations
+    struct ImplementationsResponse
+      JSON.mapping(
+        status: String,
+        message: String,
+        implementations: {type: Array(ImplementationLocation), nilable: true}
```

Comment:
just a style comment, you can also do `implementations: Array(ImplementationLocation)?` (with a `?`), no opinion on that though, as you like

Reply:
@bew Done! Thank you :+1: